service-front/.gitignore

```diff
@@ -0,0 +1,8 @@
+/.phpcs-cache
+/.phpunit.cache
+/.psalm-cache
+vendor/
+config/development.config.php
+data/cache/*
+!data/cache/.gitkeep
```

Comment:
Since we're storing the cache in `/tmp/cache`, we don't use these directories so don't need to include them in the ignore file